data-processing-lib/python/src/data_processing/data_access/data_access_s3.py

```diff
@@ -73,17 +71,31 @@ def __init__(
         self.n_samples = n_samples
         self.files_to_use = files_to_use
 
+        logger.debug(f"S3 access key provided: {access_key}")
+        logger.debug(f"S3 secret key provided: no soup for you!")
+        logger.debug(f"S3 region {region}")
+        logger.debug(f"S3 endpoint/url: {endpoint}")
+        logger.debug(f"S input folder: {self.input_folder}")
+        logger.debug(f"S3 output folder: {self.output_folder}")
+        logger.debug(f"S3 data sets: {self.d_sets}")
+        logger.debug(f"S3 checkpoint: {self.checkpoint}")
+        logger.debug(f"S3 m_files: {self.m_files}")
+        logger.debug(f"S3 n_samples: {self.n_samples}")
+        logger.debug(f"S3 files_to_use: {self.files_to_use}")
+
+        self.arrS3 = ArrowS3(access_key, secret_key, endpoint=endpoint, region=region)
+
     def get_access_key(self):
-        self.s3_credentials.get("access_key", None),
+        return self.s3_credentials.get("access_key", None)
 
     def get_secret_key(self):
-        return (self.s3_credentials.get("secret_key", None),)
+        return self.s3_credentials.get("secret_key", None)
 
     def get_endpoint(self):
-        return (self.s3_credentials.get("url", None),)
+        return self.s3_credentials.get("url", None)
 
     def get_region(self):
-        return (self.s3_credentials.get("region", None),)
+        return self.s3_credentials.get("region", None)
```

Comment:
Sorry, these are completely artificials. It was fine before. I do not see the need for these

Reply:
they are need for spark